packages/ripple-nuxt-tide/test/unit/__snapshots__/markup-transpiler.test.js.snap

```diff
@@ -31,11 +31,11 @@ exports[`Markup transpiler should able transpile html into vue template with plu
 </tbody></table></div>
 
 <!-- pluginEmbeddedMediaVideo -->
-<rpl-embedded-video width=\\"854\\" height=\\"480\\" src=\\"https://www.youtube.com/embed/Xb4ElZqit8k?autoplay=0&amp;start=0&amp;rel=0\\" class=\\"rpl-markup__embedded-video\\" variant=\\"link\\" :display-transcript=\\"true\\" :media-link=\\"embeddedMediaVideoData0.mediaLink\\" transcript=\\"Some video caption text &lt;&amp;&apos;&quot;\\"></rpl-embedded-video>
+<rpl-embedded-video width=\\"854\\" height=\\"480\\" src=\\"https://www.youtube.com/embed/Xb4ElZqit8k?autoplay=0&amp;start=0&amp;rel=0\\" title class=\\"rpl-markup__embedded-video\\" variant=\\"link\\" :display-transcript=\\"true\\" :media-link=\\"embeddedMediaVideoData0.mediaLink\\" transcript=\\"Some video caption text &lt;&amp;&apos;&quot;\\"></rpl-embedded-video>
```

Comment:
It's still not quite right - this top half is run with plugins, while the second half is without. So the test run _with_ plugins isn't rendering the title. I can see in the plugins code you're using `element.attr('title')` but the fixture has title on the iframe itself, ie. `iframe.attr('title')`.